### PR TITLE
Add user_filter & gitlab_default_theme configuration option

### PR DESCRIPTION
**Pull request by svenhartmann**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@ gitlab_ldap_method: "plain"
 gitlab_ldap_bind_dn: "CN=Username,CN=Users,DC=example,DC=com"
 gitlab_ldap_password: "password"
 gitlab_ldap_base: "DC=example,DC=com"
+gitlab_user_filter: '(employeeType=developer)'
```
Comment (nbeernink) [inline defaults/main.yml]: Perhaps best to keep this default an empty string?

**Comment by nbeernink** [comment]
The travis build does not seem to be failing due to this patch but more because of an installation issue.

**Comment by nbeernink** [comment]
Can this be re-tested somehow? 

The test failed due to:
> "WARNING: The following packages cannot be authenticated!", "  gitlab-ce"

**Comment by geerlingguy** [comment]
Since the merging of #105, these extra settings, which are slightly less common, could be set via that PR's new variable.

**Comment by nbeernink** [comment]
Thanks Jeff, that makes sense. Probably best to always handle uncommon options that way. :)

**Pull request closed without merging**